src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -217,22 +227,23 @@ struct H5WorldtubeBoundaryDataManager : db::SimpleTag {
   using option_tags =
       tmpl::list<OptionTags::LMax, OptionTags::BoundaryDataFilename,
                  OptionTags::H5LookaheadTimes, OptionTags::H5Interpolator,
-                 OptionTags::H5IsBondiData>;
+                 OptionTags::H5IsBondiData, OptionTags::FixSpecNormalization>;
 
   static constexpr bool pass_metavariables = false;
   static type create_from_options(
       const size_t l_max, const std::string& filename,
       const size_t number_of_lookahead_times,
       const std::unique_ptr<intrp::SpanInterpolator>& interpolator,
-      const bool h5_is_bondi_data) noexcept {
+      const bool h5_is_bondi_data, const bool h5_spec_normalization) noexcept {
```

Comment:
Is `h5_spec_normalization` the bool read in from `OptionTags::FixSpecNormalization`? If so, why not name it `fix_spec_normalization` for consistency?